content/en/docs/tasks/administer-cluster/kubelet-config-file.md

```diff
@@ -36,13 +36,21 @@ port: 20250
 serializeImagePulls: false
 evictionHard:
     memory.available:  "200Mi"
+    nodefs.available:  "10%"
+    imagefs.available: "15%"
+    nodefs.inodesFree: "5%"
 ```
 
-In the example, the Kubelet is configured to serve on IP address 192.168.0.8 and port 20250, pull images in parallel,
-and evict Pods when available memory drops below 200Mi. Since only one of the four evictionHard thresholds is configured,
-other evictionHard thresholds are reset to 0 from their built-in defaults.
-All other Kubelet configuration values are left at their built-in defaults, unless overridden
-by flags. Command line flags which target the same value as a config file will override that value.
+In this example, the Kubelet is configured with the following settings:
+
+1. address: The Kubelet will serve on IP address 192.168.0.8.
+2. port: The Kubelet will serve on port 20250.
+3. serializeImagePulls: Image pulls will be done in parallel.
```

Comment:
You could either:

- write something more like:
  ```markdown
  * `address` and `port`: The kubelet listens on TCP port 20250, binding to the IPv4 address 192.168.0.8 only.
  * `serializeImagePulls`: The kubelet may pull images in parallel. Because `maxParallelImagePulls` is
     not set, there is no limit on how many parallel pulls can happen at one time.
  * `evictionSoft`: Set some eviction thresholds.
  ...more details here...
  * `evictionHard`: Set more eviction thresholds.
  ```
* add similar information to the example YAML snippet, using YAML comments.
* both of the above